hns/hns_test.go

```diff
@@ -16,10 +17,20 @@ import (
 )
 
 var netAdapter string
+var controllerAddr string
+var controllerPort int
+var useActualController bool
 
 func init() {
 	flag.StringVar(&netAdapter, "netAdapter", "Ethernet0",
 		"Network adapter to connect HNS switch to")
+	flag.StringVar(&controllerAddr, "controllerAddr",
+		"10.7.0.54", "Contrail controller addr")
```

Comment:
Are you sure that you want to have hard-code ip address? What when this test will be run on jenkins?